fem/ceed_algebraic.hpp

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
```

Comment:
Is there a specific reason for this file to be in the `fem` directory instead of `fem/libceed`?

Reply:
Originally the stuff in `fem/libceed` was more C-style, libceed-style stuff that I was considering contributing to libCEED but this file was always going to be an MFEM interface. All of that has changed a lot and I am open to moving this.

Reply:
Okay, I think it might indeed be cleaner to move it under `fem/libceed` in this case.